config/locales/models/groupings/en.yml

```diff
@@ -3,4 +3,6 @@ en:
   activerecord:
     attributes:
       grouping:
+        inactive_groups: inactive group(s)
```

Comment:
For newer strings, we're trying to prefer using `count` to avoid the awkward `(s)`. See https://github.com/fnando/i18n-js/tree/v3#pluralization for details